lib/shipit.rb

```diff
@@ -140,6 +140,14 @@ def host
     secrets.host.presence
   end
 
+  def enforce_publish_config
```

Comment:
@casperisfine Is this approach acceptable?

Assuming this is correct, the next step would be to update `shipit` with these corresponding secrets.